.kitchen.yml

```diff
@@ -8,16 +8,21 @@ driver:
 platforms:
 - name: centos-6
   driver:
-    box: chef/centos-6.6
+    box: chef/centos-6.6  # vagrant
+    image: centos-6-5-x64 # digitalocean
```

Comment:
I def would have preferred to keep them in the separate file, and spent a bit of time trying to get that to work in another cookbook. But platforms don't get merged between `.kitchen.yml` and `kitchen.local.yml`, so there is some unexpected behaviour when you add `run_list` entries to the `.kitchen.yml` (because you have to duplicate every platform entry customization into the local file).

It seemed like a confusing layer of indirection that would bite someone later, so I just opted for mixing digitalocean config into the main file for sanity :)

Your call though. I know it probably offends your sense of order, but I think it's clear to any reader, so that seemed best to me!